L1Trigger/L1TTrackMatch/python/L1TrackerEtMissProducer_cfi.py

```diff
@@ -1,12 +1,15 @@
 import FWCore.ParameterSet.Config as cms
+from L1Trigger.VertexFinder.VertexProducer_cff import VertexProducer
 
 L1TrackerEtMiss = cms.EDProducer('L1TrackerEtMissProducer',
     L1TrackInputTag = cms.InputTag("TTTracksFromTrackletEmulation", "Level1TTTracks"),
-    L1VertexInputTag = cms.InputTag("L1TkPrimaryVertex"),
+    L1VertexInputTag = cms.InputTag("VertexProducer", VertexProducer.l1VertexCollectionName.value()),
```

Comment:
Hi.

This module assumes the PV producer is called `VertexProducer`, but the L1 sequence in [SimL1Emulator_cff.py](https://github.com/cms-l1t-offline/cmssw/blob/phase2-l1t-integration-CMSSW_11_1_7/L1Trigger/Configuration/python/SimL1Emulator_cff.py#L119) currently has it as `L1VertexFinder`, and indeed when running the standard sequence I get a bunch of warnings of the sort
````
%MSG-e L1TrackerEtMissProducer:  L1TrackerEtMissProducer:L1TrackerEtMiss  17-May-2021 10:45:22 CEST Run: 1 Event: 186111

Warning: VertexCollection not found in the event. Exit
````

@rekovic @emacdonald16 @aperloff

Reply:
The change should happen in https://github.com/cms-l1t-offline/cmssw/blob/phase2-l1t-integration-CMSSW_11_1_7/L1Trigger/Configuration/python/SimL1Emulator_cff.py. All you would need would be a line like:
```python
L1TrackEtMiss.L1VertexInputTag = cms.InputTag("L1VertexFinder", VertexProducer.l1VertexCollectionName.value())
```